app/tools/formTool.js

```diff
@@ -0,0 +1,37 @@
+function formBuilder(...args) {
+  let form = '';
+  // check for data objects
+  if (args.length > 0) {
+    let i;
+    let x;
+    // through data create specified input type
+    for (i = 0; i < args.length; i += 1) {
+      if (args[i].field === 'input') {
+        form += '<label for="' + args[i].id + '" class="label">' + args[i].label + '</label>';
+        form += '<' + args[i].field + ' type="' + args[i].type + '" id="' + args[i].id + '"/>';
+      } else if (args[i].field === 'select') {
+        form += '<label for="' + args[i].id + '" class="label">' + args[i].label + '</label>';
+        form += '<' + args[i].field + ' id="' + args[i].id + '">';
+        if (args[i].options) {
+          for (x = 0; x < args[i].options.length; x += 1) {
+            form += '<option value="' + args[i].options[x] + '">' + args[i].options[x] + '</option>';
```

Comment:
Here's a great example of ```i``` and ```x``` having no semantic value. As another developer, looking at this code, I need to spend time deciphering what you have logically constructed.